pkg/operation/botanist/component/clusterautoscaler/cluster_autoscaler.go

```diff
@@ -52,9 +52,10 @@ import (
 const (
 	// ServiceName is the name of the service of the cluster-autoscaler.
 	ServiceName = "cluster-autoscaler"
+	// ContainerName is the container name of the cluster-autoscaler
+	ContainerName = v1beta1constants.DeploymentNameClusterAutoscaler
```

Comment:
Why is this exported now?

Reply:
Because it is used in the unit test. I do not see much value to keep it hardcoded in the tests. It is just a double maintenance in my opinion

Reply:
We try to avoid usages of constants in unit tests to detect if they were changes (maybe unintentionally/by accident). Please revert all related changes accordingly.